packages/client/src/users/__tests__/getTitles.test.ts

```diff
@@ -21,7 +22,9 @@ describe('getTitles', () => {
 
     fixtures.success({ response });
 
-    await expect(usersClient.getTitles(query)).resolves.toBe(response);
+    await expect(usersClient.getTitles(query as GetTitlesQuery)).resolves.toBe(
```

Comment:
Instead of "forcing" the type on query object, to be accepted, you can keep this line how it was and just change where the query object is created. For example:

```
const query: GetTitlesQuery = {
   page: 1,
   pageSize: 1,
};
```

this way, the object will always be of type `GetTitlesQuery`.